nifi-nar-bundles/nifi-standard-services/pom.xml

```diff
@@ -54,5 +54,6 @@
         <module>nifi-hadoop-dbcp-service-bundle</module>
         <module>nifi-kerberos-user-service-api</module>
         <module>nifi-kerberos-user-service-bundle</module>
+        <module>nifi-web-client-provider-bundle</module>
```

Comment:
I'm not sure if it's intentional or not, but the nifi-web-client-provider-service-nar is missing from the nifi-assembly's pom so the service doesn't appear in the UI.